labelbox/schema/webhook.py

```diff
@@ -77,9 +77,15 @@ def create(client, topics, url, secret, project) -> "Webhook":
             ValueError: If the topic is not one of Topic or status is not one of Status
 
         Information on configuring your server can be found here (this is where the url points to and the secret is set).
-                        https://docs.labelbox.com/en/configure-editor/webhooks-setup#setup-steps
+                        https://docs.labelbox.com/reference/webhook
```

Comment:
A little out of scope, but this docs link is broken.

Reply:
nice!